commands/version/__tests__/version-command.test.js

```diff
@@ -48,6 +48,9 @@ expect.addSnapshotSerializer(require("@lerna-test/serialize-git-sha"));
 
 describe("VersionCommand", () => {
   describe("normal mode", () => {
+    beforeEach(() => {
+      checkWorkingTree.mockReset();
```

Comment:
Why is this necessary? We are already [clearing mocks](https://github.com/lerna/lerna/blob/4e3f8d8087e0e2e8e83c7ea79558c517203583c2/jest.config.js#L4) after every test.

Reply:
Resetting removes mock implementations (clearing doesn't). This shouldn't have been necessary since `mockImplementationOnce` is being used, a second look revealed that I shouldn't be mocking an implementation for an error that is **not** expected. Fixed.

Reply:
Fix: https://github.com/lerna/lerna/pull/1991/commits/86a88aa19807e3d25bf815bf6a35cfff1e278869